matter/si91x/siwx917/BRD4342A/config/sl_mbedtls_config.h

```diff
@@ -112,7 +112,9 @@
 // the correct Mbed TLS / PSA Crypto options.
 #include "sli_mbedtls_config_transform_autogen.h"
 
+#if defined(SL_MBEDTLS_USE_TINYCRYPT)
```

Comment:
I'm fairly certain that this is not necessary with the changes I had done. I added a build_info.h specific to tinycript to not require this ifdef.

Reply:
See https://github.com/SiliconLabs/sdk_support/blob/main/matter/mbedtls/tinycrypt/inc/mbedtls/build_info.h

Reply:
Ok It will add based on macro `MBEDTLS_USER_CONFIG_FILE`. I will revert this change. Thanks for clarifying Mathieu.